docs/examples/hotel_reviews_qna.ipynb

```diff
@@ -0,0 +1,7258 @@
+{
```

Comment:
Awesome. Some comments on the new hotel notebook!

- Ideally refresh and run everything so the run count starts at 1 (just for aesthetics)
- Add a few markdown descriptions of your steps?
- For using `getpass` -- probably best to do a check for env var first. That way we can run these in a test suite in the future, otherwise it will hang waiting for user input :)
- Might be worth truncating some of the output because this is a pretty long example (with the printed out data).